plugins/mcreator-core/datalists/particles.yaml

```diff
@@ -92,4 +92,22 @@
 - WHITE_SMOKE
 - DUST_PLUME
 - GUST_DUST
-- TRIAL_SPAWNER_DETECTION
\ No newline at end of file
+- TRIAL_SPAWNER_DETECTION
+- BLOCK

```

Comment:
Block and some others you added are not SimpleParticleType.

MCreator only supports SimpleParticleType-s, so please reduce this list to remove non-SimpleParticleType-s

Reply:
Done